src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -6,14 +6,8 @@
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(UAPvNextTFM);$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Data.SqlClient.csproj" />
-    <HarvestIncludePaths Include="ref/net451;lib/net451;runtimes/win/lib/net451" />
-    <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
-    <HarvestIncludePaths Include="ref/netstandard1.2">
```

Comment:
Separate issue.  I bet we could delete some live configurations from this package and harvest them giving the API isn't going to change and we have a shipping package with a version of the not-supported assemblies to harvest.